plugins/woocommerce/tests/e2e-pw/tests/merchant/order-emails.spec.js

```diff
@@ -97,6 +97,68 @@ test.describe( 'Merchant > Order Action emails received', () => {
 		).toContainText( `[${ storeName }]: New order #${ newOrderId }` );
 	} );
 
+  	test( 'can receive completed email', async ( { page, baseURL } ) => {
+		// Completed order emails are sent automatically when an order's payment is completed.
+		// Verify that the email is sent, and that the content is the expected one
+		const api = new wcApi( {
+			url: baseURL,
+			consumerKey: process.env.CONSUMER_KEY,
+			consumerSecret: process.env.CONSUMER_SECRET,
+			version: 'wc/v3',
+		} );
+		await api
+			.post( 'orders', {
+				status: 'completed',
+				billing: customerBilling,
+			} )
+			.then( ( response ) => {
+				completedOrderId = response.data.id;
+			} );
+		// Search to narrow it down to just the messages we want
+		await page.goto(
+			`wp-admin/tools.php?page=wpml_plugin_log&s=${ encodeURIComponent(
+				customerBilling.email
+			) }`
+		);
+		await page.selectOption( 'select[name="search[place]"]', 'subject' );
+		await page.fill( 'input[name="search[term]"]', 'complete' );
+		await page.click( 'input#search-submit' );
+
+		// Verify that the  email has been sent
+		await expect(
+			page.getByText( `Your ${ storeName } order is now complete` )
+		).toBeVisible();
+
+		// Enter email log and select to view the content in JSON
+		await page.click( 'button[title^="View log"]' );
+		await page.locator( '#wp-mail-logging-modal-format-json' ).click();
+
+		// Verify that the message includes an order processing confirmation
+		await expect(
+			page.locator( '#wp-mail-logging-modal-content-body-content' )
+		).toContainText( 'We have finished processing your order.' );
+
+		// Verify that the email address is the correct one
+		await expect(
+			page.locator( '#wp-mail-logging-modal-content-body-content' )
+		).toContainText( customerBilling.email );
+
+		// Verify that the email contains the order ID
+		await expect(
+			page.locator( '#wp-mail-logging-modal-content-body-content' )
```

Comment:
You could const the selector at the top of this test since it has been used several times:
`const emailBodyContent = '#wp-mail-logging-modal-content-body-content';`

Reply:
Good idea!